src/Microsoft.Azure.WebJobs.ServiceBus/Config/ServiceBusExtensionConfig.cs

```diff
@@ -17,6 +18,7 @@ namespace Microsoft.Azure.WebJobs.ServiceBus.Config
     /// <summary>
     /// Extension configuration provider used to register ServiceBus triggers and binders
     /// </summary>
+    [Extension("ServiceBus")]
     public class ServiceBusExtensionConfig : IExtensionConfigProvider
```

Comment:
Rename to `ServiceBusExtensionConfigProvider` to be consistent?

Reply:
Fixed